openedx-unsupported/edx-analytics-pipeline
# SE-1069 Module Engagement: handle case when no elasticsearch host is configured

Runs just ModuleEngagementRosterPartitionTask instead of the full ModuleEngagementRosterIndexTask if no elasticsearch host is configured.

**JIRA tickets**: [OSPR-3784](https://openedx.atlassian.net/browse/OSPR-3784)

**Discussions**: https://github.com/open-craft/edx-analytics-pipeline/pull/10

**Dependencies**: None

**Merge deadline**: None

**Testing instructions**:

1. don't configure an elasticsearch host (TODO: full instructions)
2. verify that ModuleEngagementRosterPartitionTask shows in the ModuleEngagementWorkflowTask task logs, instead of ModuleEngagementRosterIndexTask

**Author notes and concerns**:



**Reviewers**

**Settings**
```yaml

```
